<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="DocWidget" name="tab">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;sclkqscncolsncoqsidzadzadzadz&lt;span style=&quot; color:#d7b4ff;&quot;&gt;zdzdaddazdzadzazda&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="DocWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnreg"/>
    <addaction name="actionEnregistrer_sous"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionCopier"/>
    <addaction name="actionCouper"/>
    <addaction name="actionColler"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnreg"/>
   <addaction name="actionEnregistrer_sous"/>
   <addaction name="actionColler"/>
   <addaction name="actionCopier"/>
   <addaction name="actionCouper"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/nouveau.jpeg</normaloff>
     <normalon>../nouveau.jpeg</normalon>:/new/prefix1/nouveau.jpeg</iconset>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/repertoire.jpeg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnreg">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/enregistrer.jpeg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/enregistrer_sous.jpeg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/copier.jpeg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Copier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/couper.jpeg</normaloff>:/new/prefix1/couper.jpeg</iconset>
   </property>
   <property name="text">
    <string>Couper</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/coller.jpeg</normaloff>:/new/prefix1/coller.jpeg</iconset>
   </property>
   <property name="text">
    <string>Coller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DocWidget</class>
   <extends>QWidget</extends>
   <header>docwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
